website/frontend/src/components/footer/footer.css

```diff
@@ -0,0 +1,11 @@
+.footer{
+    width: 100%;
+    height: 160px;
+    background-color: #343A40;
+    margin-top: 100px;
+    align-items: center;
+}
+.footer span{
```

Comment:
perhaps add a new line between each component in your `.css` files to enhance readability